.babelrc

```diff
@@ -0,0 +1 @@
+{ "presets": ["es2015"] }
```

Comment:
You'll probably want `react` in your presets here, especially if you use JSX. I'm fine either way and won't bikeshed on the [myriad](https://github.com/substack/hyperx) [alternatives](https://github.com/mlmorg/react-hyperscript).